fe/estimator.py

```diff
@@ -9,16 +9,23 @@
 
 from typing import Tuple, List, Any
 
+import dataclasses
+import jax.numpy as jnp
+
+@dataclasses.dataclass
 class SimulationResult:
-    def __init__(self, xs=None, du_dls=None, du_dps=None):
-        self.xs = xs
-        self.du_dls = du_dls
-        self.du_dps = du_dps
+   xs: np.array
```

Comment:
Should these each be `Optional[np.array]`?

Reply:
For now I don't want to make them optional. Right now we have no way of configuring the front-end code to only specify partially filled members. So for the sake of consistency I'd rather we just return everything for now and profile for size/speed issues later on.